scala3doc/test/dotty/dokka/diagram/HierarchyTest.scala

```diff
@@ -0,0 +1,98 @@
+                    assertEquals(List("E1", "E2"), x.getKnownChildrenAsStrings)
+                    val graph = MemberExtension.getFrom(x).map(_.graph)     
+                    assertEquals(true, graph.isDefined)
+                    assertEquals(
+                        Seq(
+                            "Object" -> "Any",
+                            "A1" -> "Object",
+                            "A2[Int]" -> "Object",
+                            "A3[Int, String]" -> "Object",
+                            "B1" -> "Object",
+                            "B1" -> "A1",
+                            "B2" -> "Object",
+                            "B2" -> "A1",
+                            "B2" -> "A2[Int]",
+                            "B3" -> "Object",
+                            "B3" -> "A2[Int]",
+                            "B3" -> "A3[Int, String]",
+                            "C1[A, B, C]" -> "Object",
+                            "C1[A, B, C]" -> "B1",
+                            "C1[A, B, C]" -> "B2",
+                            "C1[A, B, C]" -> "B3",
+                            "E1" -> "C1[A, B, C]",
+                            "E2" -> "C1[A, B, C]"
+                        ).sorted,
+                        graph.get.edges.map((a, b) => (a.signature.getName, b.signature.getName)).sorted
+                    )
+                }
+                if (x.getName == "E2") {
+                    assertEquals(List("A1", "A2[Int]", "A3[Int, String]", "Any", "B1", "B2", "B3", "C1[Int, Boolean, Any]", "D2[Int, Boolean]", "D3", "Object"), x.getParentsAsStrings)
+                    assertEquals(List("C1[Int, Boolean, Any]", "D2[Int, Boolean]", "D3"), x.getDirectParentsAsStrings)
+                    assertEquals(List.empty, x.getKnownChildrenAsStrings)
+                    val graph = MemberExtension.getFrom(x).map(_.graph)   
+                    assertEquals(true, graph.isDefined)
+                    assertEquals(
+                        Seq(
+                            "Object" -> "Any",
+                            // "A1" -> "Object", // These are not applicable beacuase of bug and its workaround 
+                            // "A2[Int]" -> "Object", // More info at ClassLikeSupport.scala:37
+                            // "A3[Int, String]" -> "Object",
+                            // "B1" -> "Object",
+                            // "B1" -> "A1",
+                            // "B2" -> "Object",
+                            // "B2" -> "A1",
+                            // "B2" -> "A2[Int]",
+                            // "B3" -> "Object",
+                            // "B3" -> "A2[Int]",
+                            // "B3" -> "A3[Int, String]",
+                            // "C1[Int, Boolean, Any]" -> "Object",
+                            // "C1[Int, Boolean, Any]" -> "B1",
+                            // "C1[Int, Boolean, Any]" -> "B2",
+                            // "C1[Int, Boolean, Any]" -> "B3",
```

Comment:
This doesn't need to be here, no?

Reply:
Do you point out only `C1[Int, Boolean, Any] -> B3` or all comments? If only one, I am convinced you overlooked `C1` inherits from all `B1, B2, B3`. If all, there is a problem I stated with traversing up the tree when we encounter generic type. I will look into that if I have more time, but for now the tests will not pass with them uncommented, though they are truly expected values.

Reply:
I mean the commented-out code in general.

Reply:
So what is your proposal for commented values? I though it would be convenient for somebody to uncomment them as soon as the bug will be fixed meanwhile reminding anyone who would find this code that we are aware of the missing entries, but I can cut them

Reply:
Ok, I see. In that case let's leave the comment here.